cmake/modules/FindFlatBuffers.cmake

```diff
@@ -0,0 +1,73 @@
+#
+# FLATBUFFERS_FOUND - system has FlatBuffers compiler and headers
+# FLATBUFFERS_FLATC_EXECUTABLE - the flatc compiler executable
+# FLATBUFFERS_INCLUDE_DIRS - the FlatFuffers include directory
+# FLATBUFFERS_MESSAGES_INCLUDE_DIR - the directory for generated headers
+
+if(ENABLE_INTERNAL_FLATBUFFERS)
+  include(ExternalProject)
+  file(STRINGS ${CMAKE_SOURCE_DIR}/tools/depends/native/flatbuffers-native/Makefile VER REGEX "^[ ]*VERSION[ ]*=.+$")
+  string(REGEX REPLACE "^[ ]*VERSION[ ]*=[ ]*" "" FLATBUFFERS_VER "${VER}")
+
+  # Allow user to override the download URL with a local tarball
+  # Needed for offline build envs
+  if(FLATBUFFERS_URL)
+    get_filename_component(FLATBUFFERS_URL "${FLATBUFFERS_URL}" ABSOLUTE)
+  else()
+    set(FLATBUFFERS_URL http://mirrors.kodi.tv/build-deps/sources/flatbuffers-${FLATBUFFERS_VER}.tar.gz)
+  endif()
+  if(VERBOSE)
+    message(STATUS "FLATBUFFERS_URL: ${FLATBUFFERS_URL}")
+  endif()
+
+  set(FLATBUFFERS_FLATC_EXECUTABLE ${CMAKE_BINARY_DIR}/${CORE_BUILD_DIR}/bin/flatc CACHE INTERNAL "FlatBuffer compiler")
+  set(FLATBUFFERS_INCLUDE_DIR ${CMAKE_BINARY_DIR}/${CORE_BUILD_DIR}/include CACHE INTERNAL "FlatBuffer include dir")
+  set(FLATBUFFERS_MESSAGES_INCLUDE_DIR ${CMAKE_BINARY_DIR}/${CORE_BUILD_DIR}/cores/RetroPlayer/messages CACHE INTERNAL "Generated FlatBuffer headers")
+
+  externalproject_add(flatbuffers
+                      URL ${FLATBUFFERS_URL}
+                      DOWNLOAD_DIR ${CMAKE_BINARY_DIR}/${CORE_BUILD_DIR}/download
+                      PREFIX ${CORE_BUILD_DIR}/flatbuffers
+                      CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${CMAKE_BINARY_DIR}/${CORE_BUILD_DIR}
+                                 -DCMAKE_BUILD_TYPE=Release
+                                 -DFLATBUFFERS_CODE_COVERAGE=OFF
+                                 -DFLATBUFFERS_BUILD_TESTS=OFF
+                                 -DFLATBUFFERS_INSTALL=ON
+                                 -DFLATBUFFERS_BUILD_FLATLIB=OFF
+                                 -DFLATBUFFERS_BUILD_FLATC=ON
+                                 -DFLATBUFFERS_BUILD_FLATHASH=OFF
+                                 -DFLATBUFFERS_BUILD_GRPCTEST=OFF
+                                 -DFLATBUFFERS_BUILD_SHAREDLIB=OFF
+                                 "${EXTRA_ARGS}"
+                      PATCH_COMMAND patch -p1 < ${CORE_SOURCE_DIR}/tools/depends/native/flatbuffers-native/0001-Fix-compiler-warning.patch
+                      BUILD_BYPRODUCTS ${FLATBUFFERS_FLATC_EXECUTABLE})
+  set_target_properties(flatbuffers PROPERTIES FOLDER "External Projects"
+                                    INTERFACE_INCLUDE_DIRECTORIES ${FLATBUFFERS_INCLUDE_DIR})
+else()
+  find_program(FLATBUFFERS_FLATC_EXECUTABLE NAMES flatc)
+  find_path(FLATBUFFERS_INCLUDE_DIR NAMES flatbuffers/flatbuffers.h)
+  set(FLATBUFFERS_MESSAGES_INCLUDE_DIR ${CMAKE_BINARY_DIR}/${CORE_BUILD_DIR}/cores/RetroPlayer/messages CACHE INTERNAL "Generated Flatbuffer headers")
```

Comment:
Please include the 2nd fix I posted earlier instead of doubling this line from above

Reply:
I thought I grabbed the fix you posted? What should this line be?